.gitignore

```diff
@@ -1,3 +1,2 @@
-node_modules
-*.log
-.node-version
+build/
```

Comment:
I'm pretty sure the built files need to be in the repo to be served by github pages

Reply:
Ok. We can do it another way, without this breaking changes